lib-openjpeg/src/main/java/org/esa/snap/lib/openjpeg/dataio/OpenJP2Decoder.java

```diff
@@ -273,9 +273,9 @@ private Raster decompress(Rectangle roi) throws IOException {
         while (this.pendingWrites.contains(this.tileFile)) {
             Thread.yield();
         }
-        int[] bandOffsets = new int[] {0};
-        /*int[] bandOffsets = new int[this.numBands];
-        Arrays.fill(bandOffsets,0);*/
+        //int[] bandOffsets = new int[] {0};
```

Comment:
better remove than comment-out this line

Reply:
Updates made.